### ISSUE TITLE
Do not reload commits in file history view when opening diff editor from it?

### ISSUE THREAD

**Issue by ryenus** [eamodio/vscode-gitlens#741]
- GitLens Version: 9.7.1
  
- VSCode Version: 1.33.1
- OS Version: macOS 10.14

Steps to Reproduce:

1. choose a file which had been renamed in the past
2. open the file history view
3. ensure "Follow renames" mode is turned on
3. click through the history commits to show the diff(s), no error for the latest commits
4. click an earlier commit before the renaming, the diff is shown correctly in the diff editor, but now the file history view itself failed to reload with error git ENOENT

#### Expected behavior

The error happens possibly because the file history was being reloaded with the old name before the renaming, 

Maybe using option `--follow` all the time could help? Or maybe it's unnecessary to reload the file history view when selecting the listed commits from itself?

**Comment by ryenus** [comment]
This is what happened, the file selected had been renamed in the 2nd to last commit, then the error ENOENT would happen when earlier commits were selected.

![rename ENOENT](https://user-images.githubusercontent.com/610161/57422519-f2e4ce00-7242-11e9-96c4-5ce4b10044dd.gif)

**Comment by eamodio** [comment]
@ryenus Can you turn on tracing and report back with the output? I can't seem to reproduce this.

**Comment by eamodio** [comment]
I am closing this issue because it needs more information and has not had recent activity.  Please re-open this issue when more details can be provided.

Thanks!

**Comment by ryenus** [comment]
@eamodio Hey, sorry I missed the earlier comment.

Actually, the code is from an open source repo on github.com:
https://github.com/spark-jobserver/spark-jobserver/

I just checked with the same repository and the bug seems to be already fixed with GitLens `9.8.5` on VSCode `1.36.1`.

As explained earlier, the file `SparkJob.scala` had been moved to a different directory with commit `0919588`, hence tracking the file history would be problematic, as if `git log` without the `--follow` option.

Nevertheless, it's great that GitLens is always getting better, thank you very much for the great work!
